src/rust/engine/process_execution/remote/src/remote_cache.rs

```diff
@@ -487,6 +488,27 @@ impl process_execution::CommandRunner for CommandRunner {
         let (command_digest, action_digest) =
             crate::remote::ensure_action_stored_locally(&self.store, &command, &action).await?;
 
+        workunit.update_metadata(|initial| {
+            initial.map(|(initial, level)| {
+                let mut user_metadata = initial.user_metadata;
+                user_metadata.push((
+                    "action.digest".to_string(),
```

Comment:
Right now I just included the hash values as strings in the metadata. The stdout/stderr are included as actual digest though, is it worth including these debug values as true digests? 

To do so, I'd need to add them to the WorkunitMetadata struct, either as fields or push them into the artifacts array.

Reply:
Feels like we should also log the `size_bytes`, yeah. Probably these should be true fields on WorkunitMetadata, and not user_metadata.

Reply:
I've moved them onto the WorkunitMetadata directly, let me know if this is not what you meant